packages/okta-vue/src/Auth.js

```diff
@@ -50,7 +50,7 @@ function install (Vue, options) {
       return accessToken ? oktaAuth.token.getUserInfo(accessToken) : undefined
     },
     authRedirectGuard () {
-      return async (from, to, next) => {
+      return async (to, from, next) => {
         if (from.matched.some(record => record.meta.requiresAuth) && !(await this.isAuthenticated())) {
```

Comment:
In order to keep this functionality, you'll need to update this logic to use the new variable names.

`if (from.matched...) ` **needs to be** `if (to.matched...)`

`localStorage.setItem('referrerPath', from.path || '/')`  **needs to be** `localStorage.setItem('referrerPath', to.path || '/')`

You should be able to validate this locally by running `npm test` (after updating the file `test/e2e/harness/config/dev-env.js`.